btr-web/btr-main-app/components/individual-person/AddNew.vue

```diff
@@ -41,85 +41,116 @@
+            and control of the right to elect, appoint, or remove directors of the business.
+
+            Complete following assessment to determine this person’s beneficial ownership status.
+          </p>
+        </div>
+        <div class="flex-col py-5">
+          <p class="font-bold py-5">
+            Control of Shares and Votes
+          </p>
+          <p class="text-justify">
+            Enter the percentage of shares and/or shares entitled to votes at general meetings
+            this person owns or ultimately controls (or can exercise ultimate effective control over).
+          </p>
+        </div>
+        <div class="flex-col py-5">
+          <UInput
+            v-model="percentOfShares"
+            placeholder="Percent of Shares"
+            type="text"
+            variant="none"
+            class="w-1/5 bg-gray-100 border-b-2 my-5 p-2"
+          >
+            <template #trailing>
+              <span class="text-gray-500 dark:text-gray-400 text-xs">%</span>
+            </template>
+          </UInput>
+          <UInput
+            v-model="percentOfVotes"
+            placeholder="Percent of Votes"
+            type="text"
+            variant="none"
+            class="w-1/5 bg-gray-100 border-b-2 my-5 p-2"
+          >
+            <template #trailing>
+              <span class="text-gray-500 dark:text-gray-400 text-xs">%</span>
+            </template>
+          </UInput>
+        </div>
+        <div>
+          <p class="font-bold mt-5">
+            {{ $t('labels.birthdate') }}
+          </p>
+          <BcrosInputsDateSelect class="mt-5" :max-date="maxDate" @selection="birthdate = $event" />
+        </div>
+
+        <div class="my-10">
+          <BcrosInputsAddress
+            id="addNewPersonLastKnownAddress"
+            v-model="lastKnownAddress"
+            :label="$t('labels.lastKnownAddress')"
+          />
+        </div>
+
+        <div>
+          <p class="font-bold py-5">
+            {{ $t('labels.taxNumber') }}
+          </p>
+          <p class="text-justify">
+            A Social Insurance Number (SIN), an Individual Tax Number (ITN), or a Temporary
+            Taxation Number (TTN) is required if this person has one.
+          </p>
+          <IndividualPersonTaxInfoTaxNumber
+            id="addNewPersonTaxNumber"
+            v-model="taxInfoModel"
+            name="taxNumber"
+            :label="$t('labels.taxNumber')"
+            data-cy="testTaxNumber"
+          />
+        </div>
+
+        <div>
+          <p class="font-bold py-5">
+            {{ $t('labels.taxResidency') }}
+          </p>
+          <p class="text-justify">
+            Indicate if this person is a resident of Canada for Income Tax purposes.
+          </p>
+          <IndividualPersonTaxInfoTaxResidency
+            id="addNewPersonTaxResidency"
+            v-model="isTaxResident"
+          />
+        </div>
+      </template>
+    </UForm>
```

Comment:
@hfekete @kialj876  Since the taxNumber will be validated by zod in the same schema as full name and email, they need to be wrapped in the same <UForm> tag. I moved the </UForm> closing tag to the end. 

The problem is, now the entire form is in one UForm. When we enter a percentage of share or click the radio button for citizenship, the zod validation will be triggered, and error message for full name, preferred name, email, and tax number will show up if there is any.

Reply:
Figured it out. Just ignore the above comment for now.